api/apps/geoprocessing/src/marxan-sandboxed-runner/adapters-blm/blm-input-files.factory.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+import { InputFilesFs } from '../adapters-single/scenario-data/input-files-fs';
+import { WorkspaceBuilder } from '../ports/workspace-builder';
+import { AssetFactory } from './asset-factory.service';
+import { BlmInputFiles } from './blm-input-files';
+
+@Injectable()
+export class BlmInputFilesFactory {
```

Comment:
Needed for creating `BlmInputFiles` instance per blm-calibration run. Single `BlmInputFiles` as a provider would be potentially shared between several blm-calibration runs, meaning that if any of those runs is cancelled every `Workspace` would be removed (including those belonging to the rest of blm-calibration runs)